### DISCUSSION TITLE
手动执行提示配置文件不存在

### DISCUSSION THREAD

**Discussion by mumuability** [0-8-4/miui-auto-tasks#110] [问与答｜Q&A] [answered]
我应该把配置文件放到哪里呢，放在项目文件夹检测不到

**Comment by TardisXE** [comment]
配置文件“config.yaml”应与脚本文件“miuitask.py”处于同一目录下

**Reply by mumuability** [reply]
[已经这样做了]
![image](https://github.com/0-8-4/miui-auto-tasks/assets/142678994/ef6e425b-7de4-4684-b2a9-da35ebbda60c)

**Reply by TardisXE** [reply]
能否提供程序运行后的输出内容呢

**Reply by mumuability** [reply]
![image](https://github.com/0-8-4/miui-auto-tasks/assets/142678994/d43c96ba-32f5-4776-805c-8cae6317dbcf)

**Reply by 0-8-4** [reply]
请 提供一下 终端/cmd/powershell/命令行 的 执行目录 （不是miuitask.py 的存放目录）

**Reply by mumuability** [reply]
我没有通过终端来执行，手动下载了miuitask.py这个文件，然后创建配置文件

**Comment by 0-8-4** [comment, marked as answer]
那其他的文件呢？比如utils

**Reply by mumuability** [reply]
代码中所需的模块都已经下载，utils下载后仍然提示缺失，于是我把项目中的utils文件放到了模块utils文件夹内，此时能够正常运行

**Reply by mumuability** [reply]
![image](https://github.com/0-8-4/miui-auto-tasks/assets/142678994/85a74db7-74be-4ab8-be08-2306f3dbd0c5)

**Reply by TardisXE** [reply]
utils.py应该存在于和miuitask.py、config.yaml在同一个文件夹内的utils子文件夹

**Reply by mumuability** [reply]
成功了！谢谢
